plv8_func.cc

```diff
@@ -376,19 +382,20 @@ plv8_FunctionInvoker(const FunctionCallbackInfo<v8::Value> &args) throw()
 		FlushErrorState();
 		FreeErrorData(edata);
 
-		Handle<v8::Object> err = Exception::Error(message)->ToObject(isolate->GetCurrentContext()).ToLocalChecked();
-		err->Set(String::NewFromUtf8(isolate, "sqlerrcode"), sqlerrcode);
+		Local<v8::Object> err = Exception::Error(message).As<Object>();
+		err->Set(context, String::NewFromUtf8(isolate, "sqlerrcode").ToLocalChecked(), sqlerrcode).ToChecked();
+        err->Set(context, String::NewFromUtf8(isolate, "sqlerrcode").ToLocalChecked(), sqlerrcode).Check();
 #if PG_VERSION_NUM >= 90300
-		err->Set(String::NewFromUtf8(isolate, "schema_name"), schema_name);
-		err->Set(String::NewFromUtf8(isolate, "table_name"), table_name);
-		err->Set(String::NewFromUtf8(isolate, "column_name"), column_name);
-		err->Set(String::NewFromUtf8(isolate, "datatype_name"), datatype_name);
-		err->Set(String::NewFromUtf8(isolate, "constraint_name"), constraint_name);
-		err->Set(String::NewFromUtf8(isolate, "detail"), detail);
-		err->Set(String::NewFromUtf8(isolate, "hint"), hint);
-		err->Set(String::NewFromUtf8(isolate, "context"), context);
-		err->Set(String::NewFromUtf8(isolate, "internalquery"), internalquery);
-		err->Set(String::NewFromUtf8(isolate, "code"), code);
+        err->Set(context, String::NewFromUtf8(isolate, "schema_name").ToLocalChecked(), schema_name).Check();
+        err->Set(context, String::NewFromUtf8(isolate, "table_name").ToLocalChecked(), table_name).Check();
+        err->Set(context, String::NewFromUtf8(isolate, "column_name").ToLocalChecked(), column_name).Check();
+        err->Set(context, String::NewFromUtf8(isolate, "datatype_name").ToLocalChecked(), datatype_name).Check();
+        err->Set(context, String::NewFromUtf8(isolate, "constraint_name").ToLocalChecked(), constraint_name).Check();
+        err->Set(context, String::NewFromUtf8(isolate, "detail").ToLocalChecked(), detail).Check();
+        err->Set(context, String::NewFromUtf8(isolate, "hint").ToLocalChecked(), hint).Check();
+        err->Set(context, String::NewFromUtf8(isolate, "context").ToLocalChecked(), err_context).Check();
+        err->Set(context, String::NewFromUtf8(isolate, "internalquery").ToLocalChecked(), internalquery).Check();
+        err->Set(context, String::NewFromUtf8(isolate, "code").ToLocalChecked(), code).Check();
```

Comment:
this, I think, is my most frustrating of all - no consistent use of `context`, `isolate`, or lack thereof (not your fault, just my frustration at v8)

Reply:
Streamlined with `NewFromUtf8Literal`, still ugly, but less...